gui/src/2D/advancedDynamicTexture.ts

```diff
@@ -594,7 +594,18 @@ export class AdvancedDynamicTexture extends DynamicTexture {
     /** @hidden */
     public _getGlobalViewport(): Viewport {
         var size = this.getSize();
-        return this._fullscreenViewport.toGlobal(size.width, size.height);
+        const globalViewPort = this._fullscreenViewport.toGlobal(size.width, size.height);
+
+        const targetX = globalViewPort.width * (1 / this.rootContainer.scaleX);
+        const targetY = globalViewPort.height * (1 / this.rootContainer.scaleY);
```

Comment:
Please consider Math.round() because of JS floating point issues (you may end up with `Float` width/height, but they should be `Integer`)

Reply:
Opening an issue to investigate at a later time. 
https://github.com/BabylonJS/Babylon.js/issues/11539